copier.yaml

```diff
@@ -56,18 +86,37 @@ use_python_pytest:
 
 use_python_pip_audit:
     type: bool
+    when: "{{ preset_name == 'production' }}"
     help: Do you want to include Python dependency vulnerability scanning (via pip-audit)?
     default: yes
 
 use_github_actions:
     type: bool
+    when: "{{ preset_name == 'production' }}"
     help: Do you want to include Github Actions workflows for build and testnet deployment?
-    default: yes
+    default: |-
+        {% if preset_name == 'production' -%}
+            yes
+        {%- else -%}
+            no
+        {%- endif %}
 
 use_pre_commit:
     type: bool
+    when: "{{ preset_name == 'production' }}"
     help: Do you want to include pre-commit for linting, type checking and formatting?
-    default: yes
+    default: |-
+        {% if preset_name == 'production' -%}
+            yes
+        {%- else -%}
+            no
+        {%- endif %}
+
+use_dispenser:
+    type: bool
+    when: "{{ preset_name == 'production' }}"
+    help: Do you want to fund your deployment account using an optional dispenser account?

```

Comment:
Random thought, should we include a link to a Markdown page describing the difference?